source/connect.txt

```diff
@@ -169,3 +37,12 @@ the active instance and returns the Compass view to the initial
 connection dialog.
```

Comment:
I don't think we need "To disconnect from a MongoDB instance in Compass" -- the title/page make this pretty obvious in my opinion. Thoughts?

Reply:
The following text is rendering above the screenshot:

> In the dialog, enter the name of the index to delete.

(Should not be there)